docs/apache-airflow/alembic_revisions.rst

```diff
@@ -0,0 +1,332 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+Alembic Revisions
+=================
+
+The following table shows the list of alembic revisions
+
+
+.. list-table:: Alembic Revisions(until Airflow 2.0.0):
+   :widths: 20 20 20 40
+   :header-rows: 1
+
+   * - Revision ID
+     - Revises ID
+     - Airflow Version
+     - Summary
+   * - e3a246e0dc1
+     - 1507a7289a2f
+     - 2.0.0
+     - create is_encrypted
+   * - 1507a7289a2f
+     - 13eb55f81627
+     - 2.0.0
+     - maintain history for compatibility with earlier migrations
+   * - 13eb55f81627
+     - 338e90f54d61
+     - 2.0.0
```

Comment:
All the migrations have version `2.0.0` which is incorrect. If you check the example I have in https://github.com/apache/airflow/issues/11989 -- we want to show which migration was added in which Version example 1.10.10, 1.10.12 etc